content/applications/inventory_and_mrp/inventory/product_management/product_tracking/create_sn.rst

```diff
@@ -0,0 +1,193 @@
+=====================
+Assign serial numbers
+=====================
+
+Assigning serial numbers to individual products allows for tracking of properties, :doc:`expiration
+dates <expiration_dates>`, and location throughout the supply chain, particularly benefiting
+manufacturers providing after-sales services.
+
+.. seealso::
+   - `Odoo Tutorials: Serial Numbers <https://www.youtube.com/watch?v=ZP-gMz2X5AY>`_
+   - :doc:`serial_numbers`
+
+In Odoo, serial numbers are assigned to products:
+
+- in the :ref:`Detailed Operations page <inventory/product_management/detailed-operations>` on a
+  receipt
+- by clicking the :ref:`Assign Serial Numbers <inventory/product_management/assign-sn>` button on a
+  receipt
+- in the :ref:`Open: Stock move window <inventory/product_management/stock-move-section>` on a
+  receipt
+- when closing a manufacturing order
+- when making an inventory adjustment
+
+.. _inventory/product_management/detailed-operations-popup:
+
+.. figure:: create_sn/assign-serial-numbers.png
+   :align: center
+   :alt: Display the Detailed Operations smart button and bulleted list icon on a receipt.
+
+   Display the **Detailed Operations** smart button and bulleted list icon on a receipt.
+
+Configuration
+=============
+
+To assign serial numbers to products, activate the :guilabel:`Lots and Serial Numbers` feature in
+:menuselection:`Inventory app --> Configuration --> Settings`, and enable :guilabel:`Tracking by
+Unique Serial Number` in the :guilabel:`Inventory` tab of the item's product form.
+
+.. seealso::
+   - :ref:`Enable serial numbers <inventory/product_management/enable-lots>`
+   - :ref:`Track products by serial numbers <inventory/product_management/configure-lots>`
+
+.. _inventory/product_management/configure-new-serials:
+
+Next, enable creating new serial numbers by going to :menuselection:`Inventory app --> Configuration
+--> Operations Types`.
+
+Select the desired operation type (e.g. *Receipts*, *Delivery Orders* or *Manufacturing*) and select
+the :guilabel:`Create New` option in the :guilabel:`Lots/Serial Numbers` section of the operation
+type's configuration page.
```

Comment:
```suggestion
From the :guilabel:`Operations Types` page, select the desired operation type (e.g. :guilabel:`Receipts`, :guilabel:`Delivery Orders`, or :guilabel:`Manufacturing`), and select the :guilabel:`Create New` option in the :guilabel:`Lots/Serial Numbers` section of the operation type's configuration page.
```
suggestion GUIs here because, at this point in the doc's instructions, the user will be *on* the 'Operations Type' page, with those parenthetical options *actually* appearing on the screen/UI